processor/src/processor.py

```
@@ -56,39 +58,80 @@ def get_next_task(token: str) -> QueueTask:
-			client.table(settings.queue_table).update({'is_processing': True}).eq('id', task.id).execute()
-
+		# Verify token
+		user = verify_token(token)
+		if not user:
+			raise AuthenticationError('Invalid token', token=token, task_id=task.id)
+
+		# Load dataset
+		try:
+			with use_client(token) as client:
+				response = client.table(settings.datasets_table).select('*').eq('id', task.dataset_id).execute()
+				dataset = Dataset(**response.data[0])
```

Ruff (0.8.0)

Local variable `dataset` is assigned to but never used

Remove assignment to unused variable `dataset`

(F841)